src/settings/ServicePointDetails.js

```diff
@@ -207,7 +197,13 @@ class ServicePointDetails extends React.Component {
                   xs={11}
                   data-test-next-period-item-label
                 >
-                  {this.state.currentPeriod.startDate + ' - ' + this.state.currentPeriod.endDate + ' (' + this.state.currentPeriod.name + ')'}
+                  {this.state.currentPeriod.startDate}
```

Comment:
We have test error here "AssertionError: expected '7/25/2021-8/29/2021 (test)' to equal '07/25/2021 - 08/29/2021 (test)'"